cp3pt0-deployment/common/utils.sh

```diff
@@ -1184,14 +1184,18 @@ function update_operator() {
     fi
 
     title "Updating ${sub_name} in namesapce ${ns}..."
+
+    # there is currently a bug with oc apply where the removed labels are added back by
+    # something (either Open Shift or OLM), so need to use patch
+    # "~1" is the escape sequence for "/" character
+    if [ ! -z "$remove_opreq_label" ]; then
+        ${OC} patch sub ${sub_name} --type='json' -p='[{"op": "remove", "path": "/metadata/labels/operator.ibm.com~1opreq-control"}]' || info "Could not patch Subscription to remove label"
```

Comment:
@bitscuit 
Would we add a namespace and use full resource name for the command
```suggestion
        ${OC} patch subscription.operators.coreos.com ${sub_name} -n ${ns} --type='json' -p='[{"op": "remove", "path": "/metadata/labels/operator.ibm.com~1opreq-control"}]' || warning "Could not patch Subscription ${sub_name} in ${ns} to remove label"
```

Reply:
Yes, updated